geometry_msgs/PoseStamped pose
---
memoria/ErrorMsg error